### ISSUE TITLE
Error starting TAK Windows Service

### ISSUE THREAD

**Issue by rcaudill** [Tapawingo/FreeTakServer#31]
Disclaimer: I'm new here so I apologize if I've missed something. Please let me know if you need more information. Thanks!

### Summary

When I try to run the Windows service (beginProgramAsService.py) I'm never prompted for an IP/port and I get the following error:

```
PS C:\Users\rcaud\Downloads\TAKfreeServerV0.6\TAKfreeServer\TAKWinService> python .\beginProgramAsService.py install
Traceback (most recent call last):
  File ".\beginProgramAsService.py", line 17, in <module>
    ThreadedServer()
  File "C:\Users\rcaud\Downloads\TAKfreeServerV0.6\TAKfreeServer\server.py", line 56, in __init__
    self.sock.bind((self.host, self.port))
socket.gaierror: [Errno 11001] getaddrinfo failed
PS C:\Users\rcaud\Downloads\TAKfreeServerV0.6\TAKfreeServer\TAKWinService>
```

### Environment Info

- Windows 10 Pro 1909
- Clean python install 
```
PS C:\Users\rcaud\Downloads\TAKfreeServerV0.6\TAKfreeServer\TAKWinService> python --version
Python 3.8.2
PS C:\Users\rcaud\Downloads\TAKfreeServerV0.6\TAKfreeServer\TAKWinService> pip install pywin32
Requirement already satisfied: pywin32 in c:\users\rcaud\appdata\local\programs\python\python38-32\lib\site-packages (227)
```
- Download latest FTS release`https://github.com/Tapawingo/TAKlib/releases/download/V0.6/TAKfreeServerV0.6.zip`

**Comment by brothercorvo** [comment]
fixed in 06.5 needs to be tested

**Comment by brothercorvo** [comment]
Deprecated function
